cmake/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.27)
+project(libbfio)
+
+add_library(libbfio STATIC
+../libbfio/libbfio_error.c
+../libbfio/libbfio_file.c
+../libbfio/libbfio_file_io_handle.c
+../libbfio/libbfio_file_pool.c
+../libbfio/libbfio_file_range.c
+../libbfio/libbfio_file_range_io_handle.c
+../libbfio/libbfio_handle.c
+../libbfio/libbfio_memory_range.c
+../libbfio/libbfio_memory_range_io_handle.c
+../libbfio/libbfio_pool.c
+../libbfio/libbfio_support.c
+../libbfio/libbfio_system_string.c)
+
+target_include_directories(libbfio PRIVATE 
```

Comment:
This is not portable, this expects dependencies to be installed in the same directory as the libbfio source